File: application/src/main/kotlin/br/com/multiproject/Post.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package br.com.multiproject;

import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

@Table("posts")
class Post {

    @Column("title")
    private String title;

    @Column("content")
    private String content;
    
}

File: application/src/main/kotlin/br/com/multiproject/Post.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package br.com.multiproject;

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column
import org.springframework.data.relational.core.mapping.Table

@Table("posts")
data class Post(    @Id
                   val id : Long,



                    @Column val title: String,@Column val content : String)
